targets/python/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import PlayFabErrors
+import PlayFabHTTP
+import json
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, customData, extraHeaders, postProcessingFunc):
```

Comment:
Reorder parameters:
request, postProcessingFunc, customData = None, extraHeaders = None
customData and extraHeaders are optional, and must come after the callback.
Let's just rename postProcessingFunc as callback, to be consistent with other SDKs.